platform/pulp/platform/fields.py

```diff
@@ -0,0 +1,80 @@
+"""
```

Comment:
What do you think about having this in `pulp.platform.models.fields` perhaps? It's for the models layer so I figured it would be in there somewhere.